srcpkgs/fractal/template

```diff
@@ -1,25 +1,29 @@
 # Template file for 'fractal'
 pkgname=fractal
-version=4.4.2
-revision=2
+version=5
+revision=1
+# FIXME: Build fails on non 64-bit archs
+# Try again once this patch makes it into a release and into fractal
+# https://gitlab.freedesktop.org/pipewire/pipewire-rs/-/merge_requests/203
```

Comment:
try this: https://github.com/void-linux/void-packages/pull/46320/files#diff-a96c43938bfb4a64d9cc96f3b023ec7f3e0338bfba1c589fb04bc4e53ff67df4R19-R22